.github/ISSUE_TEMPLATE/new_metapackage.yml

```diff
@@ -58,30 +58,32 @@ body:
         - Delphi
         - Disassemblers
         - dotNet
+        - Documents
         - Evasion
         - Exploitation
+        - File Information
         - Forensic
         - Hex Editors
         - Information Gathering
         - InnoSetup
         - Java & Android
         - Javascript
         - Lateral Movement
+        - Memory
         - Networking
-        - Office
         - Packers
         - Password Attacks
         - Payload Development
-        - PDF
         - PE
         - Persistence
         - PowerShell
         - Privilege Escalation
-        - Python
+        - Productivity Tools
         - Reconnaissance
-        - Text Editors
+        - Registry
+        - Shellcode
         - Utilities
-        - VB
+        - Visual Basic
```

Comment:
do we only have one tool in this category? thinking if we are planing to add more tools to it or if we should merge it with other categories :thinking:

Reply:
I think we might only have one, but I do think it's a good category due to it being a useful tool and I wouldn't know where else to go to look for it if I was needing help with VB type stuff. 😅